mathesar_ui/src/i18n/languages/en/dict.json

```
@@ -483,7 +483,7 @@
   "properties": "Properties",
   "provide_details_connect_existing_database": "Provide the details to connect an existing PostgreSQL database to Mathesar",
   "provide_url_to_file": "Provide a URL to the file",
-  "public_schema_info": "Every PostgreSQL database includes the \"public\" schema. This protected schema can be read by anybody who accesses the database.",
+  "public_schema_info": "Every PostgreSQL database includes the protected \"public\" schema, which can be read by any user. By default, only the database owner has the privilege to write to the schema. In PostgreSQL versions prior to 15, any user can write to the public schema.",
```

More accurate to say:

```suggestion
  "public_schema_info": "Every PostgreSQL database includes the protected \"public\" schema, which can be read by any user. By default, only the database owner has the privilege to create objects in the schema. In PostgreSQL versions prior to 15, any user can create objects in the public schema.",
```

---

"write to" makes it seem like users can't edit data or write to tables without being granted permissions. This isn't exactly _inaccurate_, but that's a misleading belief since that's true for tables in all schemas and isn't specific to the `public` schema. The specific permission that's changed is `CREATE`.